src/backends/github/API.js

```diff
@@ -67,6 +80,9 @@ export default class API {
       if (contentType && contentType.match(/json/)) {
         return this.parseJsonResponse(response);
       }
+      if (responseStatus !== 200) {
```

Comment:
That's leftover from the collaborator check. I switched to a write permission check, so I'll just remove that actually.